redbot/core/core_commands.py

```diff
@@ -739,18 +738,46 @@ class Core(commands.Cog, CoreLogic):
     @_set.command()
     @checks.guildowner()
     @commands.guild_only()
-    async def adminrole(self, ctx: commands.Context, *, role: discord.Role):
-        """Sets the admin role for this server"""
-        await ctx.bot.db.guild(ctx.guild).admin_role.set(role.id)
-        await ctx.send(_("The admin role for this guild has been set."))
+    async def addadminrole(self, ctx: commands.Context, *, role: discord.Role):
+        """ Adds an admin role for this guild """
+        async with ctx.bot.db.guild(ctx.guild).admin_role() as roles:
+            if role.id in roles:
+                return await ctx.send(_("This is already an admin role."))
+            roles.append(role.id)
+        await ctx.send(_("That role was added as an admin role for this guild."))
 
     @_set.command()
     @checks.guildowner()
     @commands.guild_only()
-    async def modrole(self, ctx: commands.Context, *, role: discord.Role):
-        """Sets the mod role for this server"""
-        await ctx.bot.db.guild(ctx.guild).mod_role.set(role.id)
-        await ctx.send(_("The mod role for this guild has been set."))
+    async def addmodrole(self, ctx: commands.Context, *, role: discord.Role):
+        """ Adds a mod role for this guild """
+        async with ctx.bot.db.guild(ctx.guild).mod_role() as roles:
+            if role.id in roles:
+                return await ctx.send(_("This is already a mod role."))
+            roles.append(role.id)
+        await ctx.send(_("That role was added as a mod role for this guild."))
+
+    @_set.command(aliases=["remadmindrole"])
+    @checks.guildowner()
+    @commands.guild_only()
+    async def removeadminrole(self, ctx: commands.Context, *, role: discord.Role):
+        """ Removes an admin role for this guild """
+        async with ctx.bot.db.guild(ctx.guild).admin_role() as roles:
+            if role.id not in roles:
+                return await ctx.send(_("This was not an admin role."))
```

Comment:
This line and L778 probably should be "That role was already not a[n] <admin/mod> role".